rhino.yml

```diff
@@ -0,0 +1,195 @@
+
+######## NOTE: To make steering sluggish, reduce steering_max_rate i.e. 0 < steering_max_rate < 2  
+
+    # Web-application for viewing vehicle data.
+    vehicle-view:
+        container_name: opendlv-vehicle-view
+        image: chalmersrevere/opendlv-vehicle-view-multi:v0.0.43
+        restart: always
+        network_mode: "host"
+        volumes:
+        - ~/recordings:/opt/vehicle-view/recordings
+        - /var/run/docker.sock:/var/run/docker.sock
+        environment:
+        - OD4SESSION_CID=111
+        - OPENDLV_VEHICLE_VIEW_PLATFORM=Rhino        
+        ports:
+        - "8081:8081"
+       
+    # Web-application for viewing signals.
+    signal-viewer:
+        image: chalmersrevere/opendlv-signal-viewer-multi:v0.0.8
+        restart: always
+        network_mode: "host"
+        ports:
+        - "8080:8080"
+        command: "--cid=111"
+
+######################################################################################
+#
+# Other Microservices.
+#
+######################################################################################
+
+    # Encoder for camera-opencv(h264) for industrial applications
+#    video-h264-encoder-amd64:
+#        build:
+#           context: https://github.com/chalmers-revere/opendlv-video-h264-encoder.git
+#            dockerfile: Dockerfile.amd64
+#        restart: on-failure
+#        network_mode: "host"
+#        ipc: "host"
+#        volumes:
+#        - /tmp:/tmp
+#       command: "--cid=111 --name=video0.i420 --width=640 --height=480"
+
+    # Encoder for camera-opencv(vp8)
+#    video-vpx-encoder:
+#        image: chalmersrevere/opendlv-video-vpx-encoder-multi:v0.0.5
+#        restart: on-failure
+#        network_mode: "host"
+#        ipc: "host"
+#        volumes:
+#        - /tmp:/tmp
+#        command: "--cid=111 --name=video0.i420 --width=640 --height=480 --vp8"
+
+    # Accessing a video stream from Video4Linux-supported cameras.
+#    dev-camera-v4l:
+#        image: chalmersrevere/opendlv-device-camera-v4l-multi:v0.0.6
+#        restart: always
+#        ipc: "host"
+#        network_mode: "host"
+#        volumes:
+#        - /tmp:/tmp
+#        command: "--camera=/dev/video0 --width=640 --height=480 --freq=20"
+
+    # Trimble GPS
+#    dev-gps-nmea:
+#        image: chalmersrevere/opendlv-device-gps-nmea-multi:v0.0.9
+#        restart: on-failure
+#        network_mode: "host"
+#        command: "--nmea_ip=10.42.42.23 --nmea_port=9999 --cid=111"
+
+    # Velodyne VLP16 Lidar
+#    dev-lidar-vlp16c:
+#        image: chalmersrevere/opendlv-device-lidar-vlp16-multi:v0.0.9
+#        restart: on-failure
+#        network_mode: "host"
+#        command: "--vlp16_ip=0.0.0.0 --vlp16_port=2368 --cid=111"
+        
+    # Convert video data into images.
+#	rec2csv_png:

```

Comment:
The reason I kept it here is because of the concatenation thing you mentioned previously with different .odvd (ofcourse it's possible with VV as well) and if i wished to run this microservice in a script to convert multiple files at the same time without manually doing it or waiting for 1 file to finish. What do you think?

Reply:
Okay for me